libs/ui/toggle/brain/src/lib/brn-toggle-group.component.ts

```diff
@@ -89,6 +90,12 @@ export class BrnToggleGroupComponent<T = unknown> implements ControlValueAccesso
 	/** The toggles within the group. */
 	private readonly _toggleButtons = contentChildren(BrnToggleDirective, { descendants: true });
 
+	ngAfterViewInit() {
+		// This is needed because we need access to our toggles through
+		// contentChildren to sync initial values set in writeValue before they are available, e.g. default values in a form
+		this.updateSelectionState();
```

Comment:
I don't like the imperative calls to this update function. 
I think I found a solution to this problem, which makes the toggle components even cleaner.
One downside is, that the state of the brn-toggle is more coupled to the group when inside a toggle-group.
I added a draft PR #484 LMKWYT